### PR TITLE
Release 2.20.0

### PR DESCRIPTION
**Pull request by thejandroman**
@Faisal78123 brought up that 2.20.0 is not present in stable. This PR brings in the `release-2.20.0` branch from https://github.com/github/backup-utils/pull/564 to stable.

### CODE REVIEW

File: bin/ghe-host-check [hunk 1]
```diff
@@ -113,7 +113,7 @@ fi
 
 # backup-utils 2.13 onwards limits support to the current and previous two releases
 # of GitHub Enterprise Server.
-supported_minimum_version="2.17.0"
+supported_minimum_version="2.18.0"
```
Comment (thejandroman) [inline bin/ghe-host-check]: Should this be 2.20?

Comment (lildude) [reply bin/ghe-host-check]: Nope. We support N-2 releases like we do for GHES upgrades as we need to support migrations.

**Comment by thejandroman** [comment]
@github/ghes-infrastructure can you approve this PR for merging if it is correct?

Possibly related to https://github.com/github/backup-utils/pull/543 and https://github.com/github/backup-utils/issues/568.

**Comment by cainejette** [comment]
`release-2.20.0` has been merged to `master` in https://github.com/github/backup-utils/pull/564. This PR I think can be abandoned since `master` will be merged to `stable` as part of the normal release flow.

**Pull request closed without merging**